packages/tables/styleguide.config.js

```diff
@@ -5,61 +5,69 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+const path = require('path');
+const reactDocgenTypescript = require('react-docgen-typescript');
+const reactDocgen = require('react-docgen');
+
 /**
  * Package specific styleguide configuration
  * https://github.com/styleguidist/react-styleguidist/blob/master/docs/Configuration.md
  */
 module.exports = {
+  propsParser: reactDocgenTypescript.withCustomConfig(
+    path.resolve(__dirname, '../../tsconfig.json'),
+    {
+      propFilter: props => {
+        return props.parent.fileName.indexOf('node_modules') === -1;
+      }
+    }
+  ).parse,
+  resolver: reactDocgen.resolver.findAllComponentDefinitions,
   sections: [
     {
       name: '',
       content: '../../packages/tables/README.md'
     },
     {
-      name: 'Tables',
-      content: '../../packages/tables/src/examples/tables.md',
+      name: 'Examples',
```

Comment:
This is an open ended question. I noticed that early on, CSS-in-JS example sections are organized like this:

```
Examples
├── Basic
└── Advanced
Elements
├── Element 1
├── Element 2
└── Element 3
```

Should the examples for `react-tables` be organized under "Basic" and "Advanced" too? Or is `react-tables` complex enough to require a different organization strategy? When would one know _not_ to follow the above organization (e.g. organizing under a "Basic", "Advanced", etc)?

Reply:
I've included the basic styling/format props as knobs in the first basic example. All of the other examples include a lot of custom logic and styling for their use-case.

Reply:
Additionally, having the separated example naming helps with readability. If I remember correctly the "Virtual scrolling" example is the most shared deep-link across all the documentation.

Reply:
Thanks for the clarification!